lib/bolt/cli.rb

```diff
@@ -134,8 +134,9 @@ def create_option_parser(results)
                 "(Optional, defaults to 100)") do |concurrency|
           results[:concurrency] = concurrency
         end
-        opts.on('--modules MODULES', "Path to modules directory") do |modules|
-          results[:modules] = modules
+        opts.on('--modulepath MODULES',
+                "List of directories containing modules") do |modulepath|
```

Comment:
Could you mention that it's separated/delimited by `#{File.PATH_SEPARATOR}` so it shows the correct value if the controller is windows or not?